css/components/badge.css

```diff
@@ -0,0 +1,155 @@
+.status-badge {
+  border-radius: 50%;
+  bottom: 0;
+  right: 0;
+  position: absolute;
+  background-color: var(--status-offline-color);
+  border: 2px solid var(--white);
+}
+
+.status-badge_xs {
+  width: 1.2rem;
+  height: 1.2rem;
+}
+
+.status-badge_s {
+  width: 1.6rem;
+  height: 1.6rem;
+}
+
+.status-badge_m {
+  width: 2.2rem;
+  height: 2.2rem;
+}
+
+.status-badge_l {
+  width: 2.4rem;
+  height: 2.4rem;
+}
+
+.status-badge_xl {
+  width: 3rem;
+  height: 3rem;
+}
+
+.status-badge_offline {
+  background-color: var(--status-offline-color);
+}
+
+.status-badge_online {
+  background-color: var(--status-available-color);
+}
+
+.status-badge_away {
+  background-color: var(--status-away-color);
+}
+
+.status-badge_busy {
+  background-color: var(--status-busy-color);
+}
+
+.badge-container {
+  position: relative;
+}
+
+.text-badge {
+  border-radius: 0.5rem;
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--white);
+  background-color: var(--demo-color-1);
+}
+
+.text-badge_s {
+  top: -10px;
+  right: -50px;
```

Comment:
Instead of rems we are using px because of positioning of the badge right or is there some other reason?

Reply:
Hello! Yes, Since the values were really small, I used px instead of rems for positioning